Sources/Xcodeproj/pbxproj().swift

```diff
@@ -94,7 +94,7 @@ public func pbxproj(srcroot: String, projectRoot: String, xcodeprojPath: String,
         print("            buildPhases = (\(module.compilePhaseReference), \(module.linkPhaseReference));")
         print("            buildRules = ();")
         print("            dependencies = (\(module.nativeTargetDependencies));")
-        print("            name = \(module.name);")
+        print("            name = \(module.c99name);")
```

Comment:
yeah, see the testcase in this commit and SR-1444 also: 
for a target A-B swift build creates module name is AB where as Xcode as A_B. I am not sure which one should be correct but using c99name for target names solves the problem in Xcode.